docker-compose-proxy.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.3'
```

Comment:
Oh sweet, I didn't know they did away with DABs, awesome!

Re: that readme, thank you.  I think someone should make a PR/issue... 🤔 

HMMMMM